romancal/datamodels/schemas/reference_files/referencefile.schema.yaml

```diff
@@ -1,7 +1,6 @@
 %YAML 1.1
 ---
-$schema: "http://stsci.edu/schemas/fits-schema/fits-schema"
```

Comment:
Like core.schema, this file can use either of the YAML or ASDF metaschemas.

Reply:
Done.